core/src/main/java/com/google/zxing/aztec/decoder/Decoder.java

```diff
@@ -155,6 +155,14 @@ private static String getEncodedData(boolean[] correctedBits) throws FormatExcep
           index += 3;
           switch (n) {
             case 0:
+              //  flush bytes before outputting FNC1
+              try {
```

Comment:
How about just pulling this block up before the switch statement? it happens in the 0 and default case anyway, and 7 leads to an exception anyway. (and just keep 2-space indent)

Reply:
Yes, this makes total sense. For 7 we don't know what to do anyway, so whatever is right.

Reply:
https://app.codecov.io/gh/zxing/zxing/commit/be2c5bdd883f9f42e794e4b0e7e1b30f436fc0a7

Reply:
This commit fc50fca compared to be2c5bd